components/ActivityCardPreview/index.js

```diff
@@ -22,14 +21,19 @@ export default function ActivityCardPreview({ activity, onToggleBookmark }) {
           width={400}
           height={360}
           alt={title}
+          style={imageStyle}
         />
       </Link>
-
+      <h2>{title}</h2>
       <StyledCateogriesUl>
         {filterCategories.map((category) => (
           <StyledCategory key={category.id}>{category.name}</StyledCategory>
         ))}
       </StyledCateogriesUl>
-    </>
+    </StyledCard>
   );
 }
+
+const imageStyle = {
```

Comment:
Nice idea 👍. But to be consistent in your styling please create a styled component instead of a style object.